digits/dataset/job.py

```diff
@@ -31,6 +31,9 @@ def json_dict(self, verbose=False):
                                       "val_count":   t.val_count,
                                       "test_count":  t.test_count,
                                       } for t in self.parse_folder_tasks()],
+                'CreateDbTasks':    [{"db_name":            t.db_name,
+                                      "create_db_log_file": t.path(t.create_db_log_file),
+                                     } for t in self.create_db_tasks()],
```

Comment:
Can we just remove this section entirely for now? I'm thinking we might add a "verbose" option later which gives more info (and paths to all relevant files), but I don't want to add this and have to remove it later and break backwards-compatibility.